examples/device-b.py

```diff
@@ -0,0 +1,18 @@
+from time import sleep, time
+
+from robocluster import Device
+
+device = Device('device-b', '224.0.0.64:32464')
+
+@device.task
+async def hello():
```

Comment:
Can these `async` keywords be removed as they are wrapped now? Or could this just be an example that emphisizes that robocluster uses the asyncio library, so you can write asyncio stuff?

Reply:
`async` needs to be used for anything that could make a blocking call, like sleeping or other I/O operations.